pkg/gemspec/parse_test.go

```diff
@@ -0,0 +1,64 @@
+package gemspec
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func TestParse(t *testing.T) {
+	vectors := []struct {
+		file      string // Test input file
+		libraries []types.Library
+	}{
+		{
+			file:      "testdata/normal00.gemspec",
+			libraries: GemspecNormal,
+		},
+		{
+			file:      "testdata/normal01.gemspec",
+			libraries: GemspecNormal,
+		},
+		{
+			file:      "testdata/normal02.gemspec",
+			libraries: GemspecNormal,
+		},
+		{
+			file:      "testdata/malformed00.gemspec",
+			libraries: GemspecMalformed,
+		},
+		{
+			file:      "testdata/malformed01.gemspec",
+			libraries: GemspecMalformed,
+		},
+	}
+
+	for _, v := range vectors {
+		t.Run(path.Base(v.file), func(t *testing.T) {
+			f, err := os.Open(v.file)
+			if err != nil {
+				t.Fatalf("Open() error: %v", err)
+			}
```

Comment:
Recently we migrated to `stretchr/testify`.
```suggestion
			require.NoError(t, err)
```